File: app/src/main/java/com/example/simplehabitassignment/delegates/CategoriesAndProgramsResponseDelegate.java
package com.example.simplehabitassignment.delegates;


import com.example.simplehabitassignment.data.vos.CategoriesAndProgramsVO;

import java.util.List;

public interface CategoriesAndProgramsResponseDelegate extends BaseDelegate {

    void onSuccess(List<CategoriesAndProgramsVO> categoriesProgramsList);

}

File: app/src/main/java/com/example/simplehabitassignment/delegates/CategoriesAndProgramsResponseDelegate.kt
package com.example.simplehabitassignment.delegates


import com.example.simplehabitassignment.data.vos.CategoriesAndProgramsVO

interface CategoriesAndProgramsResponseDelegate : BaseDelegate {

    fun onSuccess(categoriesProgramsList: List<CategoriesAndProgramsVO>)

}
